com.sap.cloud.lm.sl.cf.process/src/main/java/com/sap/cloud/lm/sl/cf/process/steps/PrepareAppsRestartStep.java

```diff
@@ -1,33 +1,33 @@
-package com.sap.cloud.lm.sl.cf.process.steps;
```

Comment:
Check whether you've set autocrlf in your Git configuration (https://git-scm.com/book/en/v2/Customizing-Git-Git-Configuration).